contracts/contracts/vault/VaultCore.sol

```diff
@@ -188,6 +238,47 @@ contract VaultCore is VaultStorage {
         }
     }
 
+    /**
+     * @dev Burn OUSD for OUSD Meta Strategy
+     * @param _amount Amount of OUSD to burn
+     *
+     * Notice: can't use nonReentrant modifier since BaseCurveStrategy's deposit
+     * already has that modifier present
+     */
+    function redeemForStrategy(uint256 _amount)
```

Comment:
This function might be more of a "burn" than a "redeem"? There's nothing coming back from doing it, name might be confusing.

Reply:
good point, renaming